src/components/index.tsx

```diff
@@ -0,0 +1,3 @@
+import Button from "./Button/Button";
```

Comment:
Here you should not change to absolute path
If components folder is moved to another project, it'll break if paths are not set

No change needed

Reply:
What should be the alternative for absolute path?

Reply:
relative path